migrations/121_pl_courses__commit_hash__add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pl_courses ADD COLUMN IF NOT EXISTS commit_hash VARCHAR(40);
```

Comment:
Using TEXT is probably safer for when they eventually do https://lwn.net/Articles/715716/

(in postgres, VARCHAR is actually implemented as TEXT with a CONSTRAINT, so there is no speed difference)

Reply:
Ah, right, forgot they were doing that with git.